ext/ddtrace.stub.php

```diff
@@ -23,6 +23,52 @@
      */
     const DBM_PROPAGATION_FULL = UNKNOWN;
 
+    class SpanEvent implements \JsonSerializable {
+        /**
+         * SpanEvent constructor.
+         *
+         * @param string $name The event name.
+         * @param int|null $timestamp The event start time in nanoseconds, if not provided set the current Unix timestamp.
+         * @param array $attributes Optional attributes for the event.
+         */
+        public function __construct(string $name, array $attributes = [], ?int $timestamp = null) {}
+
+        /**
+         * @var string The event name
+         */
+        public string $name;
+
+        /**
+         * @var string[] $attributes
+         */
+        public array $attributes;
+
+        /**
+         * @var int The event start time in nanoseconds, if not provided set the current Unix timestamp
+         */
+        public int $timestamp;
+
+        /**
+         * @return mixed
+         */
+        public function jsonSerialize(): mixed {}
+    }
+
+    class ExceptionSpanEvent extends SpanEvent {
+        /**
+         * ExceptionSpanEvent constructor.
+         *
+         * @param \Throwable $exception exception to record.
+         * @param array $attributes Optional attributes for the event.
+         */
+        public function __construct(\Throwable $exception, array $attributes = []) {}
+        
+        /**
+         * @var \Throwable
+         */
+        private \Throwable $exception;
```

Comment:
```suggestion
        public \Throwable $exception;
```
Is there a particular reason it's private?